configs/opencv-ovms/gst_capi/launch-pipeline.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# bash_cmd="./launch-pipeline.sh 
+  # $PIPELINE_EXEC_PATH $INPUTSRC $USE_ONEVPL $RENDER_MODE $RENDER_PORTRAIT_MODE"
+# 1 - pipeline path
+# 2 - inputsrc
+# 3 - use_onevpl
+# 4 - enable rendering 
+# 5 - RENDER_PORTRAIT_MODE
+# 6 - codec_type (avc or hevc)
+
+is_avc=`gst-discoverer-1.0 "$2" | grep -i h.264 | wc -l`
+
+echo "./$1 $2 $3 $4 $5 $is_avc"
+./$1 $2 $3 $4 $5 $is_avc
```

Comment:
suggest adding assignment of $1 .. $5 to more readable local variable names. Now it is hard to read this code by just $1...$5